code/modules/mob/living/carbon/human/species/emote-synth_k9.dm

```diff
@@ -0,0 +1,31 @@
+/datum/emote/living/carbon/human/synthetic/synth_k9/New()  //K9's are blacklisted from human emotes on emote.dm, we need to not block the new emotes below
+	. = ..()
+	species_type_blacklist_typecache = null
```

Comment:
sorry, i mean, why not just set this to null (or ideally an empty list) in the definition of the emote? if i'm missing something that's fine just lmk

Reply:
I assumed (incorrectly) that the blacklist lived elsewhere and on a higher level and would need to be adjusted on creation of a new mob.  Moved it to the emote!